### ISSUE TITLE
Add appimage support

### ISSUE THREAD

**Issue by BeatLink** [butterproject/butter-desktop#718]

**Comment by streger** [comment]
Please revert to almost same idea here:
https://github.com/butterproject/butter-desktop/issues/378
and before starting ideas like this to check all issues,

**Comment by streger** [comment]
and a song for you:
https://www.youtube.com/watch?v=9gR3LFDIU4Y

**Comment by BeatLink** [comment]
Appimages were only mentioned in an off topic comment on that issue. That issue mainly was for snap, which is why i decided to create this new issue
